python-sdk/src/astro/settings.py

```diff
@@ -5,7 +5,9 @@
 from astro.constants import DEFAULT_SCHEMA
 
 SCHEMA = conf.get("astro_sdk", "sql_schema", fallback=DEFAULT_SCHEMA)
-POSTGRES_SCHEMA = conf.get("astro_sdk", "postgres_default_schema", fallback=SCHEMA)
+# Airflow hook does have option to set the postgres search path or schema's.
```

Comment:
That is not true. We can run an explicit query to set the search path. See [here](https://www.postgresql.org/docs/current/ddl-schemas.html#DDL-SCHEMAS-PATH).

Reply:
in airflow, Postgres hook can we set schema?

Reply:
right, my intention for this comment was to explain that the Postgres constructor has a way to set schema since most of our operators depend on hook initialization. But look like the root cause for CI failure may be something else that needs further investigation.